nixos/modules/services/misc/nix-daemon.nix

```diff
@@ -130,11 +130,13 @@ in
         default = false;
         description = "
           If set, Nix will perform builds in a sandboxed environment that it
-          will set up automatically for each build.  This prevents
-          impurities in builds by disallowing access to dependencies
-          outside of the Nix store. This isn't enabled by default for
-          performance. It doesn't affect derivation hashes, so changing
-          this option will not trigger a rebuild of packages.
+          will set up automatically for each build. This prevents impurities
+          in builds by disallowing access to dependencies outside of the Nix 
+          store by using network and mount namespaces in a chroot environment. 
+          This isn't enabled by default for possible performance penalties
+          resulting form running in an isolated network and mount namespace. 
```

Comment:
This could be confusing since the build is not running slower due the fact it is running a different namespace.
Instead creating a sandbox takes time itself which is especially relevant if many short-running derivations (i.e creating a configuration file in NixOS) are performed:

```
This isn't enabled by default for possible performance reason due the initial setup time of a sandbox for each build.
```

Reply:
@Mic92 OK, good, that meshes better with my world view anyway - I was confused by the original phrasing as well. - thanks!